### ISSUE TITLE
Styling

### ISSUE THREAD

**Issue by drteresavasquez** [willkotheimer/personal-bio-site#28]
@willkotheimer it may be helpful to get with one of your classmates who are good at layouts to get the styles updated in your bio site.

You can also go to a place like [themeforest](https://themeforest.net/category/site-templates/personal) to get some ideas to build.

**Comment by willkotheimer** [comment]
@drteresavasquez yes I acknowledge that. I keep hearing that I’m not great at layouts. Is there anyone you know who is good at UI? I’d be happy to ask for some help w/it.

**Comment by willkotheimer** [comment]
Also, so glad I didn’t go into the UI/UX boot camp. I love Ux but I wouldn’t be good at it, lol.
